lib/phoenix_live_view/test/upload_client.ex

```diff
@@ -294,4 +294,8 @@ defmodule Phoenix.LiveViewTest.UploadClient do
   def handle_info(:garbage_collect, state) do
     {:noreply, state}
   end
+
+  def handle_info({:socket_close, _pid, _reason}, state) do
+    {:noreply, state}
```

Comment:
If the socket closes, I believe the more reasonable thing to do would be to stop the process


```suggestion
  def handle_info({:socket_close, _pid, _reason}, state) do
    {:stop, :normal, state}
```

but I'll check check where socket_close comes from to be sure :)